app/models/document.rb

```diff
@@ -87,6 +87,17 @@ def tax_return_belongs_to_client
     errors.add(:tax_return, I18n.t("forms.errors.tax_return_belongs_to_client")) unless tax_return.blank? || tax_return.client == client
   end
 
+  def uploaded_by_name_label
+    name_label = if uploaded_by.nil?
```

Comment:
could also invert this and have this be the "else" statement (without even having to explicitly do the nil check -- just set it to auto-generated if it wasn't uploaded by a user or a client. But that's 6 of one half dozen of the other.

The indenting looks a little strange on this block to me... does rubymine highlight this to fix?It could also just be this view, but it seems like the elsif statements should be indented into the same place as the original if